bin/stashcp2/tests/test.sh

```diff
@@ -0,0 +1,4 @@
+python stashcp.py --debug user/taburaad/public/2gb_file.tar .
```

Comment:
Do I need to change anything in [this travis.yml](https://github.com/opensciencegrid/htcondor-ce/blob/master/.travis.yml) to set up my docker image? I understand the general idea but am figuring out the specifics of how to configure the docker image through your [CentOS example.](https://github.com/opensciencegrid/htcondor-ce/blob/master/tests/setup_tests.sh)